packages/frontend-bo/src/components/utils/TableFull.vue

```diff
@@ -120,8 +120,12 @@ const filteredData = computed(() => {
           ? target.findIndex((item) =>
               values.toLowerCase().includes(item.toString().toLowerCase()),
             ) === -1
-          : !target.toString().toLowerCase().includes(values.toLowerCase());
+          : !target?.toString().toLowerCase().includes(values.toLowerCase());
         return noMatch;
+      } else if (typeof rule === "object") {
+        target = Array.isArray(item[rule.obj])
+          ? item[rule.obj].map((el) => el[rule.val])
+          : item[rule.obj][rule.val];
```

Comment:
```suggestion
          : item[rule.obj]?.[rule.val];
```

plus peut etre le `?? ""`

Reply:
oui. Done